src/SFA.DAS.EmployerFeedback.Api/Startup.cs

```diff
@@ -47,6 +48,7 @@ public Startup(IConfiguration configuration, IWebHostEnvironment environment)
 
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddLogging(logging => logging.AddConsole().AddDebug());
```

Comment:
I don't recognize this - why don't we usually do this? and wouldn't this be behind a only do if development check